packages/ods-react/src/components/clipboard/src/components/clipboard/Clipboard.tsx

```diff
@@ -23,39 +27,39 @@ const Clipboard: FC<ClipboardProp> = forwardRef(({
   return (
     <VendorClipboard.Root
       className={ classNames(
-        style[ 'clipboard' ],
-        { [ style[ 'clipboard--disabled' ] ]: props.disabled },
+        style['clipboard'],
+        { [style['clipboard--disabled']]: props.disabled },
         className,
       ) }
       ref={ ref }
       value={ value?.toString() }>
       <VendorClipboard.Control asChild>
         <Input
-          className={ style[ 'clipboard__input' ] }
+          className={ style['clipboard__input'] }
           readOnly
           type={ INPUT_TYPE.text }
           value={ value }
           { ...props } />
```

Comment:
the props extend should probably be before `readOnly`, `type` and `value` to avoid override of this specific config
(BTW we'll have to to check this on all our components)